mysqld_exporter.go

```diff
@@ -80,6 +83,11 @@ var (
 		"The total number of table I/O wait events for each table and operation.",
 		[]string{"schema", "name", "operation"}, nil,
 	)
+	informationSchemaUserStatisticsDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, informationSchema, "user_statistics_total"),
+		"Per-user metrics counters for each database user",
+		[]string{"user", "counter"}, nil,
```

Comment:
I'm not sure having the counter type as a label is correct use of prometheus metrics in this case.  Labels are meant to be aggregated over, like "all users".  It doesn't make sense to aggregate over `TOTAL_CONNECTIONS` and `BYTES_SENT`.  These need to be split out into separate metrics. 

Another good example is you can't have one metric name be both a counter and a gauge.  If `concurrent_connections` was enabled below, this would be invalid:

`mysql_info_schema_user_statistics_total{user="foo",counter="concurrent_connections"}` as a gauge 
and `mysql_info_schema_user_statistics_total{user="foo",counter="total_connections"}` as a counter.